typescript-promises/typescript-promises-quiz-notes.md

```diff
@@ -0,0 +1,44 @@
+# typescript-promises-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the Event Queue model of Promises?
+
+- What are the three states a Promise can be in?
+  pending: initial state, neither fulfilled nor rejected.
+  fulfilled: meaning that the operation was completed successfully.
+  rejected: meaning that the operation failed.
+
+- How do you handle the fulfillment of a Promise?
+
+- How do you handle the rejection of a Promise?
+
```

Comment:
Missing the quiz questions! It's okay if you answer them incorrectly, we will go over in the Q&A, but you always need to submit at least something

Reply:
Updated! Completely forgot after finishing the reading!